generators/php/sdk/src/endpoint/request/WrappedEndpointRequest.ts

```diff
@@ -120,6 +134,48 @@ export class WrappedEndpointRequest extends EndpointRequest {
         writer.writeNodeStatement(this.stringify({ reference: header.valueType, name: header.name.name }));
     }
 
+    private writeBodyParameter(writer: php.Writer, valueAssignment: php.AstNode, propertyName: string): void {
+        writer.writeNodeStatement(
+            php.invokeMethod({
+                method: "add",
+                arguments_: [
+                    {
+                        name: "name",
+                        assignment: php.codeblock(`'${propertyName}'`)
+                    },
+                    {
+                        name: "value",
+                        assignment: valueAssignment
+                    }
+                ],
+                on: this.getRequestBodyArgument()
+            })
+        );
+    }
+
+    private writeMultipartPart(writer: php.Writer, paramRef: string, propertyName: string): void {
+        writer.writeNodeStatement(
+            php.invokeMethod({
+                method: "addPart",
+                arguments_: [
+                    php.invokeMethod({
+                        method: "toMultipartFormDataPart",
+                        arguments_: [php.codeblock(`'${propertyName}'`)],
+                        on: php.codeblock(paramRef)
+                    })
+                ],
+                on: this.getRequestBodyArgument()
+            })
+        );
+    }
+
+    private writeMultipartPartArray(writer: php.Writer, propertyName: string): void {
```

Comment:
nit: Let's name this `writeMultipartPartFileArray` since this is tailored to file elements (i.e. the `$file` parameter).

Reply:
done